tryingsomethingyetagain/team2120/team2120/team2120App.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  team2120App.swift
+//  team2120
+//
+//  Created by Allison Fain on 10/6/22.
+//
+
+import SwiftUI
+
+@main
+struct team2120App: App {
```

Comment:
overall good setup for needed controllers/views.

Reply:
should be a good baseline for building the app

Reply:
sounds good!